UDV-Core/examples/MAM/Demo.html

```diff
@@ -5,20 +5,15 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
-<body>
-    <div id="viewerDiv"></div>
-    <img id="logoIMU" src="../data/img/logo-imu.png"></img>
-    <img id="logoLIRIS" src="../data/img/logo-liris.png"></img>
-    <script src="../../../../dist/udvcore.js"></script>
-    <script src="../../../../node_modules/moment/moment.js"></script>
-    <script src="../../../../node_modules/three/build/three.js"></script>
-    <script src="../../../src/Externals/dat.gui/dat.gui.min.js"></script>
-    <script src="../../../../dist/itowns/examples/js/GUI/GuiTools.js"></script>
-    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
-    <script src="../../../../dist/itowns/dist/itowns.js"></script>
-    <script src="../../../../dist/itowns/dist/debug.js"></script>
-    <script src="Main.js"></script>
-    <script src="MAM-Layer.js"></script>
-    <script src="MAM-keyboardControler.js"></script>
-</body>
+  <body>
+      <div id="viewerDiv"></div>
+      <img id="logoIMU" src="../data/img/logo-imu.png"></img>
+      <img id="logoLIRIS" src="../data/img/logo-liris.png"></img>
+      <script src="../../../../dist/udvcore.js"></script>
+      <script src="../../../src/Externals/dat.gui/dat.gui.min.js"></script>
+      <script src="../../../../dist/itowns/examples/js/GUI/GuiTools.js"></script>
+      <script src="../../../../dist/itowns/dist/itowns.js"></script>
+      <script src="../../../../dist/itowns/dist/debug.js"></script>
+      <script src="MAM.js"></script>
```

Comment:
Please add a comment to explain how and when we use udvcore and itowns

Reply:
udvcore is only use to create the extras windows (help and about), itowns is required to initialize the map (view, extent and position), create the controler and generate some layer by using some function and event.

Reply:
I meant leave a comment in the code....